index.html

```diff
@@ -318,41 +318,31 @@ <h3 id="unacceptablebehavior">
           <li>Use of coded language (also known as "dog whistles") used to
           rally support for hate groups or to intimidate vulnerable groups.
           </li>
+          <li>Patronizing language or behavior:
+            <ul>
+              <li>Assuming that people or groups need concepts defined or explained to them. 
+              </li>
+              <li>Assuming that particular groups of people are technically unskilled (e.g., “So easy your grandmother could do it”).
+              </li>
+              <li>Assuming your experience or knowledge on a topic is essential to a discussion, even if it takes discussion off topic or interrupts the flow of the conversation.
```

Comment:
I think this bullet needs more work before it is included.  It sounds like it is either about: 1. dominating the conversation; or 2. having an inflated perception of the value of one's own contribution, or both.

For point 1, section 3.2 already has a bullet for: "Repeatedly interrupting or talking over someone else."  How about adding something to that bullet, such as this:

> Repeatedly interrupting or talking over someone else, or unfairly dominating the conversation.

For point 2, if someone's knowledge really is "essential to a discussion", then it cannot be "off topic", so this sounds like it is about someone have an inflated self-perception.  But I don't know how to say "Don't have an inflated perception of the value of your contribution" in a way that would be actionable guidance, so I don't have a suggestion for this one.

Also, I don't know what the phrase "interrupts the flow of the conversation" means in the context of a W3C working group.  Conversation flow in a group teleconference can often be a somewhat disjointed, because people necessarily must wait their turns to speak.  People cannot always make their desired points at the ideal moments.  Is it possible to clarify the intent of that phrase?  Maybe be more concrete?